openfisca_survey_manager/utils.py

```diff
@@ -12,59 +12,59 @@
 log = logging.getLogger(__name__)
 
 
-def inflate_parameters(parameters, inflator, base_year, last_year = None, ignore_missing_units = False):
+def inflate_parameters(parameters, inflator, base_year, last_year = None, ignore_missing_units = False,
+                       start_update_instant = None, round_ndigits = 2):
 
     if (last_year is not None) and (last_year > base_year + 1):
         for year in range(base_year + 1, last_year + 1):
-            inflate_parameters(parameters, inflator, year - 1, last_year = year, ignore_missing_units = ignore_missing_units)
+            inflate_parameters(parameters, inflator, year - 1, last_year = year, ignore_missing_units = ignore_missing_units,
```

Comment:
Is it possible to add an explanation that there is a recursive call and how it works and what is the out condition ?
I understand that we inflate n years with the inflation of first year, then n-1 with inflation of second year and so on until we reach the last year, right ?

Reply:
Yep